.github/workflows/schemaspy-rollout-dev.yaml

```diff
@@ -0,0 +1,12 @@
+name: SchemaSpy - DEV restart rollout
+
+on:
+  workflow_call:
+  workflow_dispatch:
```

Comment:
Wanted to be able to use workflow_call to have it be triggered from another workflow, and workflow_dispatch, so we can call it manually. Only needed separate files for each to allow manual calls.